projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-shipping/order-detail-shipping.component.spec.ts

```diff
@@ -58,23 +58,39 @@ const mockOrder: Order = {
   ],
 };
 
+const mockB2BOrder = {
+  ...mockOrder,
+  user: {
+    name: 'Rivers',
+  },
+  costCenter: {
+    name: 'Rustic Global',
```

Comment:
dependency on sample data

Reply:
no dependency as this is mocked, but this comes from 7765 as I mentioned the non-e2e things diff are pulled  from there, so that I can continue working on other PRs